### DISCUSSION TITLE
你好，chromium87应该使用哪个版本的可可翻译？

### DISCUSSION THREAD

**Discussion by cc19264997** [chunibyocola/sc-translator-crx#105] [Q&A] [answered]
RT：

**Comment by chunibyocola** [comment, marked as answer]
在 [Firefox 扩展商店](https://addons.mozilla.org/zh-CN/firefox/addon/sctranslator/versions/)中下载最新版至本地，更改文件后缀为 `.zip` 后拖进浏览器扩展页(`chrome://extensions/`)中。

**Reply by cc19264997** [reply]
无法加载以下来源的扩展程序： D:\Users\Temp\sctranslator-4.7.0_6428_456468492. This extension requires Google Chrome version 88 or greater.

我用的版本 87.0.4280.141（正式版本） （64 位）

**Reply by chunibyocola** [reply]
是我疏忽了，有一个软限制，需要先解压缩，用记事本的方式打开`manifest.json`，找到这一行`"minimum_chrome_version": "88",`并删除。之后重新压缩回去就可以了。
